src/server/handle-redirects.js

```diff
@@ -29,4 +31,13 @@ export default ({ server, config }) => {
     const redirectsFromFile = JSON.parse(fs.readFileSync(redirectsFile, 'utf8'))
     setRedirects(server, redirectsFromFile)
   }
+  // Set redirects from a dynamic endpoint
+  if (config.redirectsEndpoint) {
+    fetch(config.redirectsEndpoint)
+      .then(resp => resp.json())
```

Comment:
```
{"name":"FetchError","message":"invalid json response body at http://dummy.api/web/app/uploads/redirects.json reason: Unexpected end of JSON input","type":"invalid-json"}
```

It throws this regardless and carries on booting up the app anyway. Whaddya think?